src/cdk/testing/tests/cross-environment.spec.ts

```diff
@@ -37,6 +38,7 @@ export function crossEnvironmentSpecs(
   // Maybe we should introduce HarnessLoader.getActiveElement(): TestElement
   // then this 3rd parameter could get removed.
   getActiveElementId: () => Promise<string | null>,
+  includeAsyncTests: boolean = true,
```

Comment:
Can we change it to `skipAsyncTests` which defaults to `false`? (we have an aversion to default true params)

Reply:
Done. Thanks!